fuel/app/classes/materia/api/v1.php

```diff
@@ -1185,4 +1185,28 @@ static private function _get_instance_for_play_id($play_id)
 		if ( ! ($inst = Widget_Instance_Manager::get($inst_id))) throw new \HttpNotFoundException;
 		return $inst;
 	}
+
+	// Sends a notification to user requesting access to instance
+	static public function request_access(string $inst_id, int $owner_id) 
```

Comment:
This API call is unnecessary, newer API endpoints are in the API controllers. `post_request_access` is in `fuel/app/classes/controller/api/instance.php`.